pages/auth.js

```diff
@@ -1,12 +1,43 @@
 const { I } = inject();
 
 module.exports = {
-  emailField: {css: '#email_create'},
-  createButton: {css: '#SubmitCreate'},
+  emailRegisterField: { css: '#email_create' },
+  createButton: { css: '#SubmitCreate' },
+  emailInput: { css: '#email' },
+  passwordInput: { css: '#passwd' },
+  loginButton: {css: '#SubmitLogin'},
 
-  createEmail(email) {
-    I.waitForVisible(this.emailField);
-    I.fillField(this.emailField, email);
+  _waitForPageLoad() {
+    I.waitForVisible(this.emailRegisterField);
+  },
+
+  registerEmail(email) {
+    this._waitForPageLoad();
+    I.fillField(this.emailRegisterField, email);
     I.click(this.createButton);
-  }
+  },
+
+  _insertEmail(email) {
+    I.fillField(this.emailInput, email);
+  },
+
+  _insertPassword(password) {
+    I.fillField(this.passwordInput, password);
+  },
+  
+  _clickLoginButton() {
+    I.click(this.loginButton);
+  },
+
+  _checkLogin() {
```

Comment:
use `check` prefix for functions, which return `true`/`false`.
rename to `verifyLogin()`